itest/loadtest/mint_batch_test.go

```diff
@@ -164,5 +164,5 @@ func mintTest(t *testing.T, ctx context.Context, cfg *Config) {
 	})
 	require.True(t, correctOp)
 
-	itest.SyncUniverses(ctx, t, alice, bob, aliceHost, cfg.TestTimeout)
```

Comment:
I think this should actually be `bobHost` and not `aliceHost`? IIUC alice is the client trying to sync to bob.

Reply:
No, I think this is correct. It was wrong before. Bob should be the client that syncs to Alice and for that it needs Alice's host address.

Reply:
There may be more than one correct way to set it up. The old `mint` test used to have `bob` add `alice` as a federation server using `bob.AddFederationServer(... something <aliceHost> ...)`, so I was just reverting to that approach. We could probably also get away with having `alice` add `bob` as the federation server. 

Does the directionality of who adds whom matter? Alice is the node doing all the minting the way the tests are setup now.

Reply:
Directionality matters, yes. Because a sync currently is only pull. So Bob will fetch everything that Alice has. But if Bob has something that Alice doesn't I don't think that's currently pushed in the other direction.